src/app/routes/_app.issues._index.tsx

```diff
@@ -280,7 +293,7 @@ function IssuesTable({ issues }: HTMLAttributes<HTMLTableElement> & { issues: Se
                             'mr-6': column.getIsSorted() === false,
                         })}
                     >
-                        Created at
+                        Last Activity
                         {column.getIsSorted() !== false &&
```

Comment:
When clearing the cache and loading the issues index site fresh, by default it is sorted by the lastActivity column, which is how it should be. However, there is no indication that it is sorted that way, the icon at the column only appears after clicking on it.

With this addition it would make the icon visible from the beginning, indicating to the user that the columns can be sorted.

If you have a better solution please feel free to change it!

```suggestion
                        {(column.getIsSorted() !== false || (!sorting.length && column.id === 'activity')) &&
```

Reply:
Yes, I was thinking of keeping all icons visible, without state to better hint that (but then it might be not very clean). This might help with centering the headers (currently some extra padding is preventing content shift when you activate the filter).

If we touch the filters, I think I've seen another couple of places where it can be cleaned up as well.

Reply:
This was solved by setting the default sorting in [35cddca](https://github.com/42-student-council/website/pull/213/commits/35cddcac7cf0b0670fd700f1c9f1b4722041cd09)